src/vehicle.cpp

```diff
@@ -4593,6 +4644,29 @@ bool vehicle::is_rotorcraft() const
            has_sufficient_rotorlift();
 }
 
+double vehicle::lift_thrust_of_abstracted_aircraft( const bool fuelled, const bool safe ) const
+{
+
+    double engine_power_in_hp = 0.00134102 * units::to_watt( total_power( fuelled, safe ) );
```

Comment:
Ditto, why are your constants blatantly copied from vehicle::lift_thrust_of_rotorcraft? You're going to have to explain what these constants mean, and why they are being used for "abstracted aircraft".

Reply:
Same reason. It was assumed that this was required for the vehicle to fly but it turned out that the checks that restrict flight do not require this code. This is why this unneeded code is now being removed.